libkodimote/libkodimote.pro

```diff
@@ -133,3 +133,8 @@ HEADERS += libkodimote_global.h \
            addonsource.h \
            profiles.h \
            profileitem.h
+
+maemo5 {
+    target.path = /opt/libkodimote/lib
+    INSTALLS += target
+}
```

Comment:
Hmm... this shouldn't be needed. The lib is compiled and linked statically. We don't need to install it in the system.